backend-layer/main.js

```diff
@@ -0,0 +1,42 @@
+/* global chrome*/
+
+let lastTabId;
```

Comment:
it holds the last visited tab, in other words the current tab the user has open.